### PR TITLE
Fluff Trenchers

### PR DESCRIPTION
**Pull request by MichaelCourtney**
https://github.com/freeorion/freeorion/issues/3207

As no one seems to be interested in having a go at this...

### CODE REVIEW

File: default/stringtables/en.txt [hunk 1]
```diff
@@ -8910,7 +8910,14 @@ Prefer [[PT_BARREN]] planets.
 
 SP_TRENCHERS_DESC
 '''Description:
-Coming soon.
+The Trenchers are an enigma of the galaxy. Who built them? Why do they constantly dig their spiral trenches? Theorys abound: "The trenches are a rudimentary form of communication" or "The patterns are of religious importance to the Trenchers". No one really knows.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]:
```suggestion
The Trenchers are an enigma of the galaxy. Who built them? Why do they constantly dig their spiral trenches? Theories abound: "The trenches are an obscure form of communication" or "The patterns are of religious importance to the Trenchers". No one really knows.
```

File: default/stringtables/en.txt [hunk 2]
```diff
@@ -8910,7 +8910,14 @@ Prefer [[PT_BARREN]] planets.
...
+
+Homeworld:
+The Trenchers homeworld is a barren planet. It is mostly flat except for the elaborate trenches dug by its inhabitants, and a series of artificial hills constructed from the soil left over by the Trenchers digging. It is nearly devoid of plant life, with the exception of a local moss that clings to the side of the trenches.
+
+Society:
+Little is known about trencher society, they seem to show little interest in anything apart from digging trenchs and the construction of new workers. The famous ships captain and explorer Captain Bourke reports having witnessed groups of Trenchers gathering together and spinning on their wheels in what he described as "some kind of ritualistic dance". However no one has ever been able to confirm the existence of this behavior, and many experts have hypothesised that Captain Bourke was adversely effected by consuming the local flora.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]:
```suggestion
Little is known about trencher society; they seem to show little interest in anything apart from digging trenches and the construction of new workers. The explorer Bourke reported witnessing groups of Trenchers gathering together and spinning on their wheels in what he described as "some kind of ritualistic dance". However, no one has ever been able to confirm the existence of this behavior, and many experts have hypothesised that Bourke was adversely effected by consuming the local flora.
```

Comment (geoffthemedio) [inline default/stringtables/en.txt]: The description in terms of observation by an specific named outsider character seems out of place.

File: default/stringtables/en.txt (hunk 1 above)
Comment (MichaelCourtney) [reply default/stringtables/en.txt]: done

File: default/stringtables/en.txt (hunk 2 above)
Comment (MichaelCourtney) [reply default/stringtables/en.txt]: fixed spelling mistake. Rest rewritten

Comment (MichaelCourtney) [reply default/stringtables/en.txt]: I agree. Was unsure how to write it best, without the specific reference. Rewritten try this

**Comment by MichaelCourtney** [comment]
fixed "little" appearing twice in 2 sentences.

**Pull request merged**